addons/account/models/account_payment.py

```diff
@@ -156,7 +156,7 @@ def _get_default_journal(self):
         currency_field='currency_id', compute='_compute_amount_signed', tracking=True,
         help='Negative value of amount field if payment_type is outbound')
     amount_company_currency_signed = fields.Monetary(
-        currency_field='company_currency_id', compute='_compute_amount_company_currency_signed')
+        currency_field='company_currency_id', compute='_compute_amount_company_currency_signed', store=True)
```

Comment:
I doubt that Odoo can accept this change to the field.  Can you check the option of extending read_group()?